subsys/net/lib/dns/dns_sd.c

```diff
@@ -25,6 +25,7 @@
 LOG_MODULE_REGISTER(net_dns_sd, CONFIG_DNS_SD_LOG_LEVEL);
 
 const char dns_sd_empty_txt[1];
+const uint16_t dns_sd_port_zero;
```

Comment:
Shouldn't we set this to 0 here?
Also wondering whether the `dns_sd_empty_txt` is cleared somewhere in the code.

Reply:
It's set to zero as part of bss (uninitialized global variables)

Reply:
Hmm, I seem to have missed the memo that bss is initialized to zero in zephyr (as the name says it contains uninitialized vars thus was expecting `=0` here).

Reply:
Yea, bss is set to zero as part of the C standard (C89). Zephyr follows suit ;-)